flint.ui/src/views/flint/ConfigurationsRothC.vue

```diff
@@ -161,6 +168,15 @@ export default {
     }
 
     function showRothCOutputContainer() {
+      let firstRun = store.state.point.firstRun
+      if (firstRun === true) {
+        notification.error({
+          message: 'Simulation produced no result',
+          description: 'Did you forget to run the simulation first?',
+          duration: 5
+        })
+        return
+      }
```

Comment:
Target `store.state.rothc.firstRun`, like said above.

Reply:
@Crystalsage sir , Done with the changes, can you please check now?
Thanks for instructing me :)